docs/Documentation/Applications/gaussian.md

```diff
@@ -0,0 +1,106 @@
+# Running Gaussian16 Software Jobs on the Eagle System
+
+*Learn about the Gaussian16 electronic structure program and how to run Gaussian16 jobs at NREL on Eagle.*
+
+**Important Note:** To run Gaussian16, users must be a member of the Gaussian user group. To be added to the group, contact [HPC-Help](mailto:hpc-help@nrel.gov). In your email message, include your username and copy the following text agreeing not to compete with Gaussian, Inc.:
+```
+I am not actively developing applications for a competing software program, or for a project in 
+collaboration with someone who is actively developing for a competing software program. I agree 
+that Gaussian output cannot be provided to anyone actively developing for a competing software program.
+
+I agree to this statement.
+
+```
+
+## Configuration and Default Settings
+
+Eagle currently has Gaussian16 Revision B.01 and C.01 installed, and the user manual can be found at the [Gaussian website](https://gaussian.com/man). Gaussian may be configured on Eagle to run on one or more physical nodes, with or without shared memory parallelism. Distributed memory, parallel setup is taken care of automatically based on settings in the SLURM script example below. Gaussian 16 C.01 also has an GPU version, and for instructions on how to run Gaussian 16 on GPU nodes, see GitHub.
+
+Previous Gaussian 09 users sometimes may feel Gaussian 16 runs slower than Gaussian 09. That's because Gaussian G16 has changed the default accuracy into `Int=Acc2E=12 Grid=Ultrafine`, which means that individual SCF iterations will take longer with G16 than with G09. 
+
+## Batch Submission with Use of In-Memory Filesystem (Preferred Method)
+
+Gaussian jobs typically write large amounts of information to temporary scratch files. When many Gaussian jobs are running, this can put a large traffic load on the Lustre parallel filesystem. To reduce this load, we recommend putting the first 5 GB or so of scratch files into a local (on-node) in-memory filesystem called `/dev/shm`.
+
+This scratch space is set automatically by the example script below. The Gaussian input file needs the following two directives to tell the program to put read-write files first in `/dev/shm` (up to 5GB below), and to put data that exceeds 5GB into files in a directory on the `/scratch` file system. An example script for batch submission is given below: 
+
+```
```

Comment:
I think it would improve the readability of the page if the script was in a drop down box, something like this: https://docs.nersc.gov/applications/vasp/#cori-haswell